soc/arm/nordic_nrf/nrf51/power.c

```diff
@@ -1,11 +1,96 @@
 /*
```

Comment:
is there any difference between nrf51 and nrf52 implementation? Could that be common?

Reply:
There is subtle (1 line) difference. I have kept the files separate as we do not have "common" folder in `nordic_nrf` three and I would like to not introduce one. I think, that large portions of these files could be moved to global header/source (see PR comment).